packages/react/src/provider.tsx

```diff
@@ -18,8 +18,14 @@ export const LogtoProvider = ({ logtoConfig, children }: LogtoProviderProperties
     const createClient = async () => {
       try {
         const client = await LogtoClient.create(logtoConfig);
-        dispatch({ type: 'INITIALIZE', isAuthenticated: client.isAuthenticated() });
         setLogtoClient(client);
+        const isLoginRedirect = client.isLoginRedirect(window.location.href);
+        // If is login redirect, should set isLoading and isInitialized at the same time
+        dispatch({
+          type: 'INITIALIZE',
+          isAuthenticated: client.isAuthenticated(),
+          isLoading: isLoginRedirect,
+        });
       } catch (error: unknown) {
         dispatch({ type: 'ERROR', error });
```

Comment:
can error passed into the dispatch with an unknown type?
```js
if (error instance of Error) {
 dispatch({type: 'ERROR', error});
 return
}
throw error;
```

Reply:
I put this code inside reducer to avoid writing same code everytime before dispatch
```ts
if (action.type === 'ERROR') {
    const { error } = action.payload;
    if (!(error instanceof Error)) {
      throw error;
    }

    return { ...state, error, isLoading: false };
  }
```